.pre-commit-config.yaml

```diff
@@ -25,6 +25,15 @@ repos:
     rev: 23.9.1
     hooks:
       - id: black-jupyter
+        name: black-src
+        exclude: docs/
+  - repo: https://github.com/psf/black-pre-commit-mirror
+    rev: 23.9.1
```

Comment:
Can you run `pre-commit autoupdate` for newer `black` that was released today?

Also, can you remove `black` from `requirements.txt`, since now it's fully in `pre-commit`

Reply:
Alrighty! I just ran `pre-commit autoupdate` — thanks.

RE: removing `black` from `requirements.txt`, we're using poetry now to handle most of our dependencies. I will remove black from our dev deps soon... we're just working to optimize our poetry lock file (its taking a bit of time right now)